src/components/ReplyComment/hooks/useReplyComment.js

```diff
@@ -132,7 +135,6 @@ const useReplyComment = ({itemProp, indexFeed, dataFeed, updateParent, page, get
     }
     return [];
   };
-
   const showChildrenCommentView = async (itemReply) => {
     const itemParentProps = await {
       ...itemProp,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `await` keyword is used incorrectly here. It should be used with a Promise. If `itemProp` is not a Promise, remove the `await` keyword.

```diff
- const itemParentProps = await {
+ const itemParentProps = {
    ...itemProp,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->